project.godot

```diff
@@ -45,6 +45,11 @@ move_down={
 , Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":-1,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":4194322,"key_label":0,"unicode":0,"echo":false,"script":null)
 ]
 }
+pause={
```

Comment:
Why add pause??? It's not relevant to this PR. Instead of using .wav for the sound files, use .ogg format instead as it offers compression (see /Sounds folder in the repository for reference). The .wav format offers higher sound quality at the expense of higher file size.